templates/firefox-ios.xliff

```diff
@@ -1521,6 +1584,30 @@
         <source>Clear Private Data</source>
         <note>Title displayed in header of the setting panel.</note>
       </trans-unit>
+      <trans-unit id="Settings.DNT.OptionAlwaysOn">
+        <source>Always</source>
+        <note>DNT Settings option for always on</note>
+      </trans-unit>
+      <trans-unit id="Settings.DNT.OptionOnWithTP">
+        <source>Only when using Tracking Protection</source>
+        <note>DNT Settings option for only turning on when Tracking Protection is also on</note>
+      </trans-unit>
+      <trans-unit id="Settings.DNT.Title">
+        <source>Send websites a “Do Not Track” signal that you don’t want to be tracked</source>
```

Comment:
I see there's still the same apostrophe nit that Flod raised in https://github.com/mozilla-l10n/firefoxios-l10n/pull/33: "You're mixing up proper apostrophes and straight quotes in strings. This string uses ’, for example intro.Slides.Private.Title uses '."